monkey/infection_monkey/exploit/tools/helpers.py

```diff
@@ -1,22 +1,38 @@
 import logging
 import random
 import string
+from pathlib import Path
 from typing import Any, Mapping
 
 from infection_monkey.model import VictimHost
 
 logger = logging.getLogger(__name__)
 
+RAND_SUFFIX_LEN = 8
+
 
 def get_random_file_suffix() -> str:
     character_set = list(string.ascii_letters + string.digits + "_" + "-")
     # random.SystemRandom can block indefinitely in Linux
-    random_string = "".join(random.choices(character_set, k=8))  # noqa: DUO102
+    random_string = "".join(random.choices(character_set, k=RAND_SUFFIX_LEN))  # noqa: DUO102
     return random_string
 
 
 def get_agent_dest_path(host: VictimHost, options: Mapping[str, Any]) -> str:
     if host.os["type"] == "windows":
-        return options["dropper_target_path_win_64"]
+        path = Path(options["dropper_target_path_win_64"])
     else:
-        return options["dropper_target_path_linux"]
+        path = Path(options["dropper_target_path_linux"])
+
+    return str(_add_random_suffix(path))
```

Comment:
I'd prefer if we used Paths everywhere. The individual exploiters can convert to string if explicit conversion is necessary.